### PR TITLE
[Docs] translate documentation for loss function

### PR DESCRIPTION
**Pull request by willaty**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
translate documentation for loss function

## Modification
translate documentation for loss function

## Who can help? @ them here!
@zengyh1900 

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/howto/losses.md [hunk 1]
```diff
@@ -1 +1,530 @@
-# 自定义损失函数（待更新）
+# 如何设计自己的损失函数
+
+`losses` 在 `MMEditing` 中注册为 `LOSSES`。
+自定义损失类似于自定义任何其他模型。
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:4]:
```suggestion
在 MMEditing 中设计自己的损失函数，步骤和在 MMEditing 中自定义任何其他模型类似。
```

File: docs/zh_cn/howto/losses.md [hunk 2]
```diff
@@ -1 +1,530 @@
...
+本节主要是为了阐明MMEditing中损失模块的设计。
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:5]:
```suggestion
本节主要具体介绍了如何在 MMEditing 中实现自定义的损失函数。
```

File: docs/zh_cn/howto/losses.md [hunk 3]
```diff
@@ -1 +1,530 @@
...
+重要的是，在编写自己的损失模块时，您应该遵循相同的设计，
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:6]:
```suggestion
本教程建议您在实现自定义的损失函数时，应该遵循本教程相同的设计，
```

File: docs/zh_cn/howto/losses.md [hunk 4]
```diff
@@ -1 +1,530 @@
...
+这样就可以在我们的框架中采用新的损失模块而无需额外的工作。
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:7]:
```suggestion
这样在我们的框架中使用您新定义的损失函数，就不需要额外的工作。
```

File: docs/zh_cn/howto/losses.md [hunk 5]
```diff
@@ -1 +1,530 @@
...
+
+本指南包括：
+
+- [设计你自己的损失函数](#如何设计自己的损失函数)
+   - [支持损失函数介绍](#支持的损失函数介绍)
+   - [设计一个新的损失函数](#设计一个新的损失函数)
+     - [MSELoss 的一个例子](#MSELoss-的一个例子)
+     - [DiscShiftLoss 的一个例子](#DiscShiftLoss-的一个例子)
+     - [GANWithCustomizedLoss 的一个例子](#GANWithCustomizedLoss-的一个例子)
+   - [可用损失](#可用损失)
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:17]:
```suggestion
   - [可用损失函数](#可用损失函数)
```

File: docs/zh_cn/howto/losses.md [hunk 6]
```diff
@@ -1 +1,530 @@
...
+Class MSELoss(nn.Module)：
+
+     def __init__(self, loss_weight=1.0, reduction='mean', sample_wise=False):
+         # 代码可以在``mmedit/models/losses/pixelwise_loss.py``中找到
+
+     def forward(self, pred, target, weight=None, **kwargs):
+         # 代码可以在``mmedit/models/losses/pixelwise_loss.py``中找到
+```
+
+鉴于损失的定义，我们现在可以通过简单地在配置文件中定义它来使用损失：
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:64]:
```suggestion
根据这个损失函数的定义，我们现在可以通过简单地在配置文件中定义它来使用损失：
```

File: docs/zh_cn/howto/losses.md [hunk 7]
```diff
@@ -1 +1,530 @@
...
+
+```python
+pixel_loss=dict(type='MSELoss', loss_weight=1.0, reduction='mean')
+```
+
+请注意，上面的“pixel_loss”必须在模型中定义。 详情请参考`customize_models`。 与模型定制类似，为了使用您定制的损失，您需要在编写后在`mmedit/models/losses/__init__.py`中导入损失。
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:70]:
```suggestion
请注意，上面的`pixel_loss`必须在模型中定义。 详情请参考[自定义模型](./models.md)。 与自定义模型类似，为了使用您自己实现的损失函数，您需要在编写后在`mmedit/models/losses/__init__.py`中导入该损失函数。
```

File: docs/zh_cn/howto/losses.md [hunk 8]
```diff
@@ -1 +1,530 @@
...
+                    )] = losses_dict[loss_module.loss_name()] + loss_
+                else:
+                    losses_dict[loss_module.loss_name()] = loss_
+        loss, log_var = self.parse_losses(losses_dict)
+
+        return loss, log_var
+
+```
+
+在这里，`_get_disc_loss` 将帮助自动组合各种损失。
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:233]:
```suggestion
在这里，`_get_disc_loss` 将帮助自动组合各种损失函数。
```

File: docs/zh_cn/howto/losses.md [hunk 9]
```diff
@@ -1 +1,530 @@
...
+
+因此，只要用户设计相同规则的损失模块，就可以在生成模型的训练中插入任何一种损失，
+无需对模型代码进行其他修改。 您只需要在配置文件中定义 `data_info` 即可。
+
+## 可用损失
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:238]:
```suggestion
## 可用损失函数
```

File: docs/zh_cn/howto/losses.md [hunk 10]
```diff
@@ -1 +1,530 @@
...
+本指南包括：
+
+- [设计你自己的损失函数](#如何设计自己的损失函数)
+   - [支持损失函数介绍](#支持的损失函数介绍)
+   - [设计一个新的损失函数](#设计一个新的损失函数)
+     - [MSELoss 的一个例子](#MSELoss-的一个例子)
+     - [DiscShiftLoss 的一个例子](#DiscShiftLoss-的一个例子)
+     - [GANWithCustomizedLoss 的一个例子](#GANWithCustomizedLoss-的一个例子)
+   - [可用损失](#可用损失)
+     - [常规损失](#常规损失)
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:18]:
```suggestion
     - [常规损失函数](#常规损失函数)
```

File: docs/zh_cn/howto/losses.md [hunk 11]
```diff
@@ -1 +1,530 @@
...
+     - [损失组件](#损失组件)
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:19]:
```suggestion
     - [损失函数组件](#损失函数组件)
```

File: docs/zh_cn/howto/losses.md [hunk 12]
```diff
@@ -1 +1,530 @@
...
+    criterion='mse')
+
+```
+
+</td>
+  </tr>
+</tbody>
+</table>
+
+### 损失组件
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:518]:
```suggestion
### 损失函数组件
```

File: docs/zh_cn/howto/losses.md [hunk 13]
```diff
@@ -1 +1,530 @@
...
+在这里，`_get_disc_loss` 将帮助自动组合各种损失。
+
+因此，只要用户设计相同规则的损失模块，就可以在生成模型的训练中插入任何一种损失，
+无需对模型代码进行其他修改。 您只需要在配置文件中定义 `data_info` 即可。
+
+## 可用损失
+
+我们在配置中列出了可用的损失示例，如下所示。
+
+### 常规损失
```
Comment (zengyh1900) [inline docs/zh_cn/howto/losses.md:242]:
```suggestion
### 常规损失函数
```

**Comment by zengyh1900** [comment]
Please fix the lint by pre-commit hook.

**Review by zengyh1900 — APPROVED** [review]
LGTM.
